staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -238,13 +243,15 @@ func (c *AvailableConditionController) sync(key string) error {
 			return err
 		}
 		hasActiveEndpoints := false
+	outer:
```

Comment:
I think that extracting the twin for loops as a helper function would be easier to read/maintain than the labelled break. However it does seem correct.